circle.yml

```diff
@@ -59,7 +59,7 @@ test:
         parallel: true
   post:
     - .circle/docker.sh build st2bundle
-    - .circle/docker.sh build st2actionrunner st2api st2auth st2exporter st2notifier st2resultstracker st2rulesengine st2sensorcontainer
+    - .circle/docker.sh build st2actionrunner st2api st2auth st2exporter st2notifier st2resultstracker st2rulesengine st2sensorcontainer st2garbagecollector
```

Comment:
So it's new service/Docker container?
You need to add new Dockerfile here too: https://github.com/StackStorm/st2-dockerfiles

Also we need new repo in Docker hub (creating manually). I did it for you: https://hub.docker.com/r/stackstorm/st2garbagecollector/